src/views/Help/index.js

```diff
@@ -2,256 +2,263 @@ import React, { Component } from 'react';
-                addition to getting emails.
-              </li>
-              <li>
-                <strong>What if I leave an Involvement?</strong> Subscribership and Membership are
-                fluid during an academic session, and are confirmed at the conclusion of a session.
-                You can come and go over the semester, and your Membership will move to your
-                official Co-Curricular Transcript only at the end of the semester.
-              </li>
-              <li>
-                <strong>What if I am in an Involvement I don&apos;t want to be part of? </strong>
-                No problem. Contact the Involvement leaders or advisor.
-              </li>
-              <li>
-                <strong>
-                  I&apos;m an alumna or alumnus. Why do I not see the join button anymore?{' '}
-                </strong>
-                Sorry, but alumni cannot join new Involvements. You could still subscribe to
-                Involvements to receive emails!
-              </li>
-            </Typography>
+          <Grid item xs={12} lg={8}>
+            <Card className="help">
+              <CardHeader
+                className="help-title"
+                title="Gordon 360 Help"
+                titleTypographyProps={{ variant: 'h4' }}
+              />
+              <CardContent>
+                <Card>
+                  <CardHeader
+                    className="help-header"
+                    title={`LOGIN INSTRUCTIONS`}
+                    titleTypographyProps={{ variant: 'body1' }}
+                  />
+                  <CardContent>
+                    <Typography variant="body1" component="ul" style={{ textAlign: 'start' }}>
+                      <li>Please use your 'firstname.lastname' username or your Gordon email address</li>
+                      <li>Use your current Gordon College account password here as well.</li>
+                    </Typography>
+                  </CardContent>
+                </Card>
+
+                <Card>
+                  <CardHeader
+                    className="help-header"
+                    title={`SITE NAVIGATION`}
+                    titleTypographyProps={{ variant: 'body1' }}
+                  />
+                  <CardContent>
+                    <Typography variant="body1" component="ul" style={{ textAlign: 'start' }}>
+                      <li>
+                        <strong>Home</strong> &ndash; Contains a chart displaying the number of days left in
+                        the current semester, alongside the number of Christian Life &amp; Worship Credits
+                        you have earned this semester. This chart can be clicked to view the CL&amp;W events
+                        you have attended. You can also view your pending requests to join Involvements, as
+                        well as requests to join your club, if you are a leader.
```

Comment:
We should clarify, involvements and involvement requests have been moved off of the Home page (although I'm not totally sure why). Instead, the components of the Home page are the Banner (which lists things happening around campus), the Days Left (or CLW and Days Left for students) card, the Dining Plan card, and the Student News card.